src/client/utils/Intercom.tsx

```diff
@@ -47,3 +47,40 @@ export const Intercom: React.FC = () => {
 
   return null
 }
+
+export const IntercomCheckoutStyles: React.FC = () => {
+  const intercomBtnSelector = '.intercom-lightweight-app-launcher'
+  const intercomTimeout = useRef<null | NodeJS.Timeout>()
+
+  const onInteraction = () => {
+    const buttonEl: HTMLElement | null = document.querySelector(
+      intercomBtnSelector,
+    )
+    if (buttonEl !== null) {
+      if (intercomTimeout.current) {
+        clearTimeout(intercomTimeout.current)
+      }
+      buttonEl.style.opacity = '1'
+      intercomTimeout.current = setTimeout(() => {
+        buttonEl.style.opacity = '0'
+      }, 800)
+    }
+  }
+  useEffect(() => {
+    window.addEventListener('scroll', onInteraction)
+    window.addEventListener('mousemove', onInteraction)
+    // onInteraction()
+  }, [])
+
+  return (
+    <style>
+      {`
+    ${intercomBtnSelector}{
+      bottom: 87px!important;
+      transition: opacity 150ms ease-out;
+      opacity: 1;
+    }
+   `}
+    </style>
+  )
```

Comment:
I've never seen this pattern before. I think it would be better to use the built-in global styles from Emotion.